openhtf/io/user_input.py

```diff
@@ -63,7 +63,8 @@ def __init__(self):
     self._response = None
     self._cond = threading.Condition()
 
-  def DisplayPrompt(self, message, text_input=False):
+  def DisplayPrompt(self, message, text_input=False,
+                    timeout=FLAGS.prompt_timeout_s):
```

Comment:
I don't see how that gotcha applies here, since we're not creating a new instance in the function definition (as in the example) but accessing an essentially global object that has already been defined elsewhere. The pattern of using a flag's value as a kwarg's seems reasonable to me. But perhaps I'm missing something?